pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -534,7 +544,7 @@ func buildBlackHoleCluster() *v2.Cluster {
 	cluster := &v2.Cluster{
 		Name:           util.BlackHoleCluster,
 		Type:           v2.Cluster_STATIC,
-		ConnectTimeout: defaultClusterConnectTimeout,
+		ConnectTimeout: 1 * time.Second,
```

Comment:
Why ? (both changing from constant to hardcoded, and from 5 to 1 ) ?

Reply:
This is the blackhole cluster.. traffic to this goes no where. I saw no reason to define a constant for this cluster alone

Reply:
Figured, thanks.